src/shared/constants/index.ts

```diff
@@ -107,7 +107,7 @@ export const CLOUD_CHECKOUT_PATH = '/checkout'
 export const CLOUD_BILLING_PATH = '/billing'
 export const CLOUD_USAGE_PATH = '/usage'
 export const CLOUD_LOGOUT_PATH = '/logout'
-export const CLOUD_USERS_PATH = '/users'
+export const CLOUD_USERS_PATH = '/members'
```

Comment:
doesn't look like this variable is being used anywhere other than this file. I say just remove this whole line

Reply:
yeah i checked and it's not being used anywhere. Removing it altogether is a good idea